qiita_db/metadata_template/sample_template.py

```diff
@@ -189,6 +187,32 @@ def study_id(self):
         """
         return self._id
 
+    @property
+    def columns_restrictions(self):
+        """Gets the dictionary of colums required
+
+        Returns
+        -------
+        dict
+            The dict of restictions
+        """
+        return SAMPLE_TEMPLATE_COLUMNS
+
+    def can_be_updated(self, **kwargs):
+        """Gets if the template can be updated
+
+        Parameters
+        ----------
+        kwargs : ignored
+            Necessary to have in parameters to support other objects.
+
+        Returns
+        -------
+        bool
+            As this is the sample template this returns always True
+        """
+        return True
```

Comment:
HUH? This is a function that just returns true? Can you add a note why it's always true?